### ISSUE TITLE
Inpaint-and-outpaint issues, The parameters overwrite_width & overwrite_height are not effective

### ISSUE THREAD

**Issue by illusiona** [lllyasviel/Fooocus#2399]
**Describe the problem**

I have set the overwrite height and width to 1500, but I cannot find this value in the log
There is another strange phenomenon, my pictures are all 1000 * 674. But it will automatically expand to 832*1216
Can you help me find the reason in the log below?

**Full Console Log**

[INFO] Mixing image prompt and inpaint is set to True
[Task Queue] Task queue is free, start task, job_id=79fb31c3-2c5b-420c-9f72-56860f8e610f
[Parameters] Adaptive CFG = 7.0
[Parameters] Sharpness = 5.0
[Parameters] ADM Scale = 1.5 : 0.8 : 0.3
[Parameters] CFG = 7.0
[Parameters] Seed = 850541230965133579
[Fooocus] Downloading upscale models ...
[Fooocus] Downloading inpainter ...
[Inpaint] Current inpaint model is E:\Fooocus_win64\Fooocus\models\inpaint\inpaint_v26.fooocus.patch
[Fooocus] Downloading control models ...
[Fooocus] Loading control models ...
extra clip vision: ['vision_model.embeddings.position_ids']
[Parameters] Sampler = dpmpp_3m_sde_gpu - karras
[Parameters] Steps = 40 - 30
[Fooocus] Initializing ...
[Fooocus] Loading models ...
Synthetic Refiner Activated
model_type EPS
UNet ADM Dimension 2816
Using pytorch attention in VAE
Working with z of shape (1, 4, 32, 32) = 4096 dimensions.
Using pytorch attention in VAE
extra {'cond_stage_model.clip_l.text_projection', 'cond_stage_model.clip_l.logit_scale', 'cond_stage_model.clip_g.transformer.text_model.embeddings.position_ids'}
Base model loaded: E:\Fooocus_win64\Fooocus\models\checkpoints\Actual_Texture_ZY_SDXL.safetensors
Synthetic Refiner Activated
Request to load LoRAs [('qinshe.safetensors', 0.8), ('E:\Fooocus_win64\Fooocus\models\inpaint\inpaint_v26.fooocus.patch', 1.0)] for model [E:\Fooocus_win64\Fooocus\models\checkpoints\Actual_Texture_ZY_SDXL.safetensors].
Loaded LoRA [E:\Fooocus_win64\Fooocus\models\loras\qinshe.safetensors] for UNet [E:\Fooocus_win64\Fooocus\models\checkpoints\Actual_Texture_ZY_SDXL.safetensors] with 722 keys at weight 0.8.
Loaded LoRA [E:\Fooocus_win64\Fooocus\models\inpaint\inpaint_v26.fooocus.patch] for UNet [E:\Fooocus_win64\Fooocus\models\checkpoints\Actual_Texture_ZY_SDXL.safetensors] with 960 keys at weight 1.0.
Request to load LoRAs [('qinshe.safetensors', 0.8)] for model [E:\Fooocus_win64\Fooocus\models\checkpoints\Actual_Texture_ZY_SDXL.safetensors].
Loaded LoRA [E:\Fooocus_win64\Fooocus\models\loras\qinshe.safetensors] for UNet [E:\Fooocus_win64\Fooocus\models\checkpoints\Actual_Texture_ZY_SDXL.safetensors] with 722 keys at weight 0.8.
Requested to load SDXLClipModel
Loading 1 new model
[Fooocus Model Management] Moving model(s) has taken 0.31 seconds
[Fooocus] Processing prompts ...
[Fooocus] Preparing Fooocus text https://github.com/mrhan1993/Fooocus-API/issues/1 ...
[Prompt Expansion] A living room in a light luxury style, intricate, elegant, sharp focus, beautiful, highly detailed, cinematic, new bright, colorful, positive, cute, cheerful, rational, creative, color, clear, logical, pleasant, peaceful, relaxed, pure, scientific, solid, thought, ideal, perfect, pretty, innocent, artistic, rich, magnificent, shining
[Fooocus] Preparing Fooocus text https://github.com/mrhan1993/Fooocus-API/issues/2 ...
[Prompt Expansion] A living room in a light luxury style, designed tender gorgeous elegant perfect intricate detailed shining divine holy great futuristic enchanted, pretty full innocent massive glowing colossal magical imposing complex highly exposed magic lush open magnificent, bright noble extremely deep vibrant excellent directed cinematic, ambient, dynamic dramatic shiny amazing composition, charming romantic atmosphere, exalted, epic, sublime, sharp focus, awesome, very inspirational
[Fooocus] Encoding positive https://github.com/mrhan1993/Fooocus-API/issues/1 ...
[Fooocus] Encoding positive https://github.com/mrhan1993/Fooocus-API/issues/2 ...
[Fooocus] Encoding negative https://github.com/mrhan1993/Fooocus-API/issues/1 ...
[Fooocus] Encoding negative https://github.com/mrhan1993/Fooocus-API/issues/2 ...
[Fooocus] Image processing ...
Upscaling image with shape (674, 1000, 3) ...
[Fooocus] VAE Inpaint encoding ...
Requested to load AutoencoderKL
Loading 1 new model
[Fooocus] VAE encoding ...
Final resolution is (674, 1000), latent is (832, 1216).
Requested to load CLIPVisionModelWithProjection
Loading 1 new model
[Fooocus Model Management] Moving model(s) has taken 0.18 seconds

**Comment by eddyizm** [comment]
You log seems to be referencing another repo. 
Did you install from here? https://github.com/mrhan1993/Fooocus-API/

**Comment by illusiona** [comment]
> You log seems to be referencing another repo. Did you install from here? https://github.com/mrhan1993/Fooocus-API/

yes

**Comment by mashb1t** [comment]
Relates to https://github.com/mrhan1993/Fooocus-API/issues/231
